.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,27 +1,14 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: Bug report 🐞
+about: Something is broken and you have a reliable reproduction? Let us know here. For questions, please use "Question" below.
+labels: needs triage, bug
```

Comment:
Any chance we can can ask the framework to the user ?  I don't have idea how though :)

Reply:
In the **System** section below we ask users to run `npx sb@next info`, which will output the `@storybook/<framework>` package installed and that helps to know the framework, but I'm not sure if you're asking this or you mean asking users to add an issue label like `app:<framework>` 🤔

Reply:
If we want the label to be automatically set based on the issue's description we can create a simple GitHub Action that will "parse" the  **System** section to find out the SB framework (and version) used and set labels accordingly. But it would not be very relevant in cases of bugs in an addon 🤔